app/lib/rocketchat.js

```diff
@@ -520,6 +520,9 @@ const RocketChat = {
 	reportMessage(messageId) {
 		return this.sdk.post('chat.reportMessage', { messageId, description: 'Message reported by user' });
 	},
+	markMessageAsUnread(firstUnreadMessageId) {
```

Comment:
Use `Rockechat.toggleRead()` instead.
You'll have to add `firstUnreadMessageId` param.